charlib/cli/utils.py

```
@@ -14,22 +15,51 @@ def find_yaml_files(path) -> list:
+            if not quiet:
+                print(f'Skipping "{str(file)}": file does not contain a config dict')
+            continue
+        # Substitute in config keys which point to other YAML files or directories
+        config = {k: resolve_subkey(v, file.parent) for k, v in config.items()}
+        # Validate the schema
+        try:
+            config = ConfigFile.validate(config)
+            break # Exit on success
+        except SchemaError:
+            if not quiet:
+                print(f'Skipping "{str(file)}": file does not contain a valid CharLib config')
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)